src/pages/document/[id].tsx

```diff
@@ -85,6 +86,20 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     } catch (error) {}
   }
 
+  if (familyData) {
+    const allSubdivisions = await Promise.all<TGeographySubdivision[]>(
+      familyData.geographies
+        .filter((country) => country.length === 3)
+        .map(async (country) => {
+          try {
+            const { data: subDivisionResponse } = await apiClient.get(`/geographies/subdivisions/${country}`);
```

Comment:
Observation: We need a better solution than making _n_ API queries, though out of scope of this PR.